src/vm/runners/builtin_runner/ec_op.rs

```diff
@@ -40,6 +44,7 @@ impl EcOpBuiltinRunner {
             stop_ptr: None,
             included,
             instances_per_component: 1,
+            cache: Rc::new(RefCell::new(HashMap::new())),
```

Comment:
I think `RefCell<HashMap<_>>` should suffice, we only need the interior mutability, right?

Reply:
Regardless, this seems to duplicate the check that already happens in the call sites of the validation rules. AFAIR it's the memory module that has a `HashMap<Relocatable>` with all the validated addresses to avoid calling the rule more than once.